### PR TITLE
Update docs re CNI helm install

### PR DESCRIPTION
**Pull request by funkypenguin**
This PR clarifies documentation on a helm-based CNI install, highlighted in https://github.com/istio/istio/issues/43632, and complements https://github.com/istio/istio/pull/44984, which improves the UX of the istiod chart.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/additional-setup/cni/index.md [hunk 1]
```diff
@@ -91,6 +77,32 @@ This race condition is mitigated by a "detect and repair" method.
...
+
+When [installing Istio using Helm](/docs/setup/install/helm/), it's possible to install the CNI as a standalone chart, by using:
+
+    {{< text syntax=bash snip_id=install_ingressgateway >}}
+    $ kubectl create namespace istio-ingress
+    $ helm install istio-ingress istio/cni -n istio-ingress --wait
+    {{< /text >}}
+Once the CNI is successfully installed, configure the istiod sidecar-injector to use the CNI by setting `istio_cni.enabled=true` in the istiod helm chart.
+
+#### Operator-based install (deprecated)
```
Comment (zirain) [inline content/en/docs/setup/additional-setup/cni/index.md:90]: I don't install via istioctl is deprecated.

Comment (hanxiaop) [reply content/en/docs/setup/additional-setup/cni/index.md:90]: install with istioctl is not deprecated, instead maybe revise the description, or the command to just be something like `$istioctl install --set components.cni.enabled=true -y`

**Comment by windsonsea** [comment]
Check the [lint errors](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/13217/lint_istio.io/1659552287808294912) please:

```
./en/docs/setup/additional-setup/cni/index.md:86: MD031 Fenced code blocks should be surrounded by blank lines
./en/docs/setup/install/helm/index.md:145: MD012 Multiple consecutive blank lines
```

**Comment by jacob-delgado** [comment]
This will be fixed by https://github.com/istio/istio.io/pull/13265

**Pull request closed without merging**